wallet/src/de/schildbach/wallet/ui/WalletActivity.java

```diff
@@ -486,7 +496,7 @@ public void onDismiss(DialogInterface dialog) {
 
         if (ContextCompat.checkSelfPermission(this,
                 Manifest.permission.WRITE_EXTERNAL_STORAGE) == PackageManager.PERMISSION_GRANTED)
-            BackupWalletDialogFragment.show(getSupportFragmentManager());
```

Comment:
Since this line is removed: `BackupWalletDialogFragment.show(getSupportFragmentManager());` the backup dialog is not shown until onResume() is called.

Reply:
Good point @HashEngineering, I've just fixed this issue.